pages/common/stripe.md

```diff
@@ -0,0 +1,24 @@
+# stripe
+
+> Interact with your Stripe account.
+> More information: <https://github.com/stripe/stripe-cli>.
+
+- Log all activity on your account:
+
+`stripe logs tail`
+
+- Listen for webhook events for `charge.succeeded` and forward them to localhost:3000/events:
```

Comment:
Perhaps this will read better?
```suggestion
- Listen for events, filtering on events with the name `charge.succeeded` and forwarding them to localhost:3000/events:
```